librz/core/disasm.c

```diff
@@ -5274,9 +5319,38 @@ static void ds_end_line_highlight(RzDisasmState *ds) {
 	}
 }
 
-// int l is for lines
-RZ_API int rz_core_print_disasm(RzPrint *p, RzCore *core, ut64 addr, ut8 *buf, int len, int l, int invbreak, int cbytes, bool json, PJ *pj, RzAnalysisFunction *pdf) {
-	int continueoninvbreak = (len == l) && invbreak;
+/**
+ * \brief Free RzAnalysisDisasmText \p p
+ */
+RZ_API void rz_analysis_disasm_text_free(RZ_NULLABLE void *p) {
```

Comment:
```suggestion
RZ_API void rz_analysis_disasm_text_free(RzAnalysisDisasmText *p) {
```

i think we can save ourselves the RZ_NULLABLE in the free cases... It is common practice to allow NULL in all free functions.

Reply:
Add the RZ_NULLABLE anyway. i add it everywhere for the swig apis.

Reply:
👍